modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/core/components/PageRenderer/WebContentVariant.es.js

```diff
@@ -49,6 +49,12 @@ export const Column = forwardRef(
 		const defaultLanguageIdRef = useRef(defaultLanguageId);
 		const dispatch = useForm();
 
+		useEffect(() => {
+			if (history?.steps.length === 0) {
+				dispatch({type: EVENT_TYPES.HISTORY.ADD});
+			}
+		}, [history, dispatch]);
+
```

Comment:
I left it separetly as they are two different effect. The first one is about creating the inital step at the first render of the Page Component, while the other one is about reseting the History state when the default locale changes.
I would not want to combine them because of this.

Reply:
i understand your point, let's leave as it is then